### PR TITLE
Show score change on hover HEXXLE-127

### PR DESCRIPTION
**Pull request by RicardoMonteiroSimoes**

### CODE REVIEW

File: unity/Assets/Scripts/Logic/TileMap.cs
Comment (DennisBannerman) [inline unity/Assets/Scripts/Logic/TileMap.cs:18]: I remember what this originally was intended to do: if a coordinate was accessed, that "wasn't there", it would initialize a new one in order to not return null values.

Seeing as this wasn't implemented anyway and so far we haven't had any problesm with NullPointers, I think we can go forward with your change here.

Just wanted to mention it, so we might remember, if we ever run into any problems regarding this.

File: unity/Assets/Scripts/Logic/TileResolver.cs
Comment (DennisBannerman) [inline unity/Assets/Scripts/Logic/TileResolver.cs:22]: Ok, I see, how this didn't end up in NullPointers before - it was done after the voids were placed.

If you'd prefer `GetTile` didn't return null values, we can go back to the intended implemenation I mentioned earlier. Relationships to tiles of type None are implemented anyway.

File: unity/Assets/Scripts/Unity/UnityMap.cs
Comment (DennisBannerman) [inline unity/Assets/Scripts/Unity/UnityMap.cs:123]: Looks good. Suppose we need to adjust this as soon as the Hand is implemented/merged by @denadex however.

File: unity/Assets/Scripts/Unity/UnityPossiblePoints.cs
Comment (DennisBannerman) [inline unity/Assets/Scripts/Unity/UnityPossiblePoints.cs:20]: Is there a reason for using `FixedUpdate` over regular `Update`? I did a quick google search, and to me it doesn't seem like it would make a difference anywy, so I'm just curious.

**Review by DennisBannerman — APPROVED** [review]

File: unity/Assets/Scripts/Logic/TileResolver.cs
Comment (RicardoMonteiroSimoes) [reply unity/Assets/Scripts/Logic/TileResolver.cs:22]: Nah, this works fine for now. I don't see the point of generating tiles that might not be needed at all

File: unity/Assets/Scripts/Unity/UnityMap.cs
Comment (RicardoMonteiroSimoes) [reply unity/Assets/Scripts/Unity/UnityMap.cs:123]: Yes, I'd make the same suggestion, merge #30 first and then rebase, should be easier to do this way.

File: unity/Assets/Scripts/Unity/UnityPossiblePoints.cs
Comment (RicardoMonteiroSimoes) [reply unity/Assets/Scripts/Unity/UnityPossiblePoints.cs:20]: `FixedUpdate` is, according to the [documentation](https://docs.unity3d.com/ScriptReference/MonoBehaviour.FixedUpdate.html), used for physics-based calculations (or well, supposed to), as the calls are constant and not variable like `Update` that is based on fps. 

It also has a neat feature: We can wait for its update explicitely in tests. There is a problem during tests where the "score" itself would never update, which was solved using [`WaitForFixedUpdate()`](https://docs.unity3d.com/ScriptReference/WaitForFixedUpdate.html) and moving the render logic to `FixedUpdate`.

File: unity/Assets/Scripts/Unity/UnityMap.cs
Comment (DennisBannerman) [reply unity/Assets/Scripts/Unity/UnityMap.cs:123]: Your call, at first glance, I'd say, it would be easier to rebase #30 onto #29 . Haven't tried anything though.

**Pull request merged**
